.github/workflows/docker_image.yml

```diff
@@ -5,6 +5,23 @@ on:
       version:
         description: "Version of the docker image to build."
         required: true
+      chrome_version:
+        description: "Version of the chrome browser."
+        required: false
+        default: "google-chrome-stable"
+      chrome_driver_version:
+        description: "Version of the chrome driver."
+        required: false
+        default: ""
+      firefox_version:
+        description: "Version of the firefox browser."
+        required: false
+        default: "latest"
+      firefox_driver_version:
+        description: "Version of the firefox geckodriver."
+        required: false
+        default: "0.30.0"
```

Comment:
I think it would be better hardcode this in the Dockerfile, because otherwise it's hard to know which combination was used to generate the docker image. Also there is normally a compatibility range between the browser and its driver which is a bit tricky.

Reply:
Alright, I'll remove them from the yaml file.

Reply:
I removed chromedriver and geckodriver from it but kept chrome and firefox version, so that we can update browser versions in the docker image without updating the Dockerfile. It is possible that geckodriver might have a compatibility issue when firefox browser version updates, not sure how we can avoid it. (for chromedriver, I managed to automatically detect valid chrome-chromedriver version set.)

> because otherwise it's hard to know which combination was used to generate the docker image.

I think we could solve this issue by adding browser versions in the docker image tag.